tests/unit/CMakeLists.txt

```diff
@@ -2,13 +2,13 @@ include(GoogleTest)
 
 set(UNIT_TEST_DIR ${CMAKE_CURRENT_LIST_DIR})
 
-function(sacn_add_dynamic_test test_name)
+function(sacn_add_dynamic_test test_name config_dir)
```

Comment:
I think the original purpose of these functions was mainly to compile against different config file directories. So if we're adding the config directory as an argument, we might want to condense this to one function.

```cmake
function(sacn_add_dynamic_test test_name config_dir)
  set(target_name ${test_name})
  add_executable(${target_name} ${ARGN})

  target_compile_definitions(${target_name} PRIVATE SACN_HAVE_CONFIG_H)
  target_include_directories(${target_name} PRIVATE
    ${config_dir}
    ${SACN_INCLUDE}
    ${SACN_SRC}
  )
  target_link_libraries(${target_name} PRIVATE EtcPalMock gtest meekrosoft::fff)
  set_target_properties(${target_name} PROPERTIES CXX_STANDARD 17 FOLDER tests)
  gtest_discover_tests(${target_name})
endfunction()
```

And at the call site:
```cmake
sacn_add_test(test_merger_dynamic ${SACN_TEST}/configs/dynamic ${TEST_MERGER_SOURCES})
sacn_add_test(test_merger_static ${SACN_TEST}/configs/static ${TEST_MERGER_SOURCES})
```

Reply:
Personally, I liked the more specialized functions and not needing a config_dir passed in every time.

Reply:
https://github.com/ETCLabs/sACN/pull/49/commits/e0484a21780d13efd1f9f88eda66567d14eb2557 https://github.com/ETCLabs/sACN/pull/49/commits/b941230b489640ed80e2a95bba8cd582266d18da

Reply:
So why are we passing in the config path on every call to sacn_add_[dynamic|static]_test?

Reply:
We no longer are. The only time the config path gets passed in is in the call to sacn_add_test, which is only called twice for the source disabled tests.